packages/lodestar-core/test/setup.ts

```diff
@@ -0,0 +1,10 @@
+import {init} from "@chainsafe/bls";
+import {before} from "mocha";
+
+before(async function () {
+  try {
+    await init("blst-native");
+  } catch (e) {
+    console.log(e);
+  }
+});
```

Comment:
Can you use mocha hooks instead of setup file? It's currently the recommended way
Example: https://github.com/ChainSafe/lodestar/blob/master/packages/spec-test-runner/test/mocha-hooks.js
Ref: https://simplecheatsheet.com/mocha-js-root-hook-plugins-cheat-sheet/

Reply:
Cool. I've written hooks it in typescript though.